### PR TITLE
[PLAYER-3352] MultiAuduoActivity: “set an audio track” button is added

### PR DESCRIPTION
**Pull request by AlinaVoronkovaEpam**

### CODE REVIEW

File: OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/MultiAudioActivity.java [hunk 1]
```diff
@@ -27,8 +29,22 @@
...
+    setContentView(R.layout.player_skin_multiaudio_layout);
     completePlayerSetup(asked);
+
+    // Set an audio track.
+    // Notice: when you set an audio track it doesn't set as a default audio track
+    Button setTrackButton = findViewById(R.id.setTrackButton);
+    setTrackButton.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View view) {
+        if (player != null && currentAudioTrack != null) {
```
Comment (atopilskii) [inline OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/MultiAudioActivity.java]: Could you extract this actions in separate method?

File: OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/MultiAudioActivity.java [hunk 2]
```diff
@@ -94,7 +106,7 @@ private void setAudioTrack() {
       String language = "eng";
       for (AudioTrack track : audioTracks) {
         Log.d("MultiAudio activity", "MultiAudio track language is: " + track.getLanguage());
-        if (TextUtils.equals(track.getLanguage(), language)) {
+        if (TextUtils.equals(track.getLanguage(), language) && !track.equals(currentAudioTrack)) {
```
Comment (atopilskii) [inline OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/MultiAudioActivity.java]: I think track can be null her. Is it right? Could you add checking for null?

Comment (AlinaVoronkovaEpam) [reply OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/MultiAudioActivity.java]: Array of audioTracks doesn't consist of null tracks

File: OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/MultiAudioActivity.java (hunk 1 above)
Comment (AlinaVoronkovaEpam) [reply OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/MultiAudioActivity.java]: What do you mean here?

**Review by atopilskii — APPROVED** [review]

**Review by IvanSakharovskii — APPROVED** [review]

**Review by IvanSakharovskii — APPROVED** [review]

**Pull request merged**